packages/orchestration/test/exos/chain-hub.test.ts

```diff
@@ -177,3 +159,35 @@ test('toward asset info in agoricNames (#9572)', async t => {
     });
   }
 });
+
+test.serial('getChainInfoByAddress', async t => {
```

Comment:
why is this serial now? each test has its own setup. 

we should avoid serial when we can

Reply:
Without this, and the addition of the `getChainInfoByAddress` test, `toward asset info in agoricNames (#9572)` starts failing with:

```
SES_UNHANDLED_REJECTION: (Error#1)
Error#1: promise watcher must be a virtual object
    at makeError (file:///Users/0xpatrick/repos/agoric-sdk/node_modules/ses/src/error/assert.js:350:61)
    at fail (file:///Users/0xpatrick/repos/agoric-sdk/node_modules/ses/src/error/assert.js:482:20)
    at Fail (file:///Users/0xpatrick/repos/agoric-sdk/node_modules/ses/src/error/assert.js:492:39)
    at file:///Users/0xpatrick/repos/agoric-sdk/packages/swingset-liveslots/src/watchedPromises.js:216:13
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

  ─

  toward asset info in agoricNames (#9572)

  Error: Promise returned by test never resolved
```

Reply:
Ok, out of scope. Turns out `provideDurableZone` mutates global state. I'll look into it